src/wwwroot/template-editor/index.html

```diff
@@ -9,18 +9,28 @@
   <meta name="keywords" content="" />
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <!-- inject:css -->
-  <link rel="stylesheet" href="//cdn.fromdoppler.com/mseditor/v1.0.0-build306/styles/styles.min.css">
+  <link rel="stylesheet" href="//cdn.fromdoppler.com/mseditor/v1.0.0-build320/styles/styles.min.css">
```

Comment:
Remember to update all references to a build of a merged branch before merging this PR.